sympy/functions/combinatorial/factorials.py

```diff
@@ -174,7 +174,7 @@ def _eval_is_positive(self):
 
     def _eval_is_real(self):
         if ((self.args[0].is_negative and self.args[0].is_noninteger) or
-           self.args[0].is_nonnegative):
+                self.args[0].is_nonnegative):
```

Comment:
Actually, now I see that in master it was only indented 3 spaces. It should either be 4 or 8 or together with parenthesis. With 4 it causes the PEP8 error.